doc/weakly_supervised.rst

```diff
@@ -177,26 +290,73 @@ thresholding.
   overfitting. If you want to avoid that, calibrate the threshold after
   fitting, on a validation set.
 
+  >>> mmc.fit(pairs, y) # will fit the threshold automatically after fitting
+
 - **Manual**: calling `set_threshold` will set the threshold to a
   particular value.
 
+  >>> mmc.set_threshold(0.4)
+
 - **Calibration**: calling `calibrate_threshold` will calibrate the
   threshold to achieve a particular score on a validation set, the score
   being among the classical scores for classification (accuracy, f1 score...).
 
+  >>> mmc.calibrate_threshold(pairs, y)
 
 See also: `sklearn.calibration`.
 
+.. _pairs_scoring:
+
+Scoring
+-------
+
+Not only are they able to predict the label of given pairs, they can also
+return a `decision_function` for a set of pairs. It is basically the "score"
+that will be thresholded to find the prediction for the pair. In fact this
+"score" is the opposite of the distance in the new space (higher score means
+ points are similar, and lower score dissimilar).
+
+>>> mmc.decision_function([[[0.6, 1.6], [1.15, 2.75]],
+...                        [[3.2, 1.1], [5.4, 6.1]]])
+array([-0.12811124, -0.74750256])
+
+This allows to return all kinds of estimator scoring usually used in classic
```

Comment:
return --> use
estimator scoring usually used in classic classification tasks --> common scoring functions for binary classification